teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb

```diff
@@ -4,201 +4,186 @@
    "cell_type": "markdown",
```

Comment:
<ol><li>Could you show the list of the colors corresponding to the interaction type, at least write something with Markdown text to explain the color?  It even better if the name of the residues could be shown.</li><li>I also saw some interactions which are not between protein and ligand, should we remove them?</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>